### PR TITLE
Bump v1.7.0 and add first pass release notes

### PR DESCRIPTION
**Pull request by alexlyp**

### CODE REVIEW

File: app/i18n/docs/en/ReleaseNotes/v1.7.0.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+This release of Decrediton includes numerous bug fixes and refinement across
+all pages/tabs.
+
+
+* Matheusd has overhauled the security of Decrediton so that we can
+keep using electron in the future, with a decent amount of assurance that it's
+safe and not prone to intrusion.  The overal jist of the work done could be
```
Comment (matheusd) [inline app/i18n/docs/en/ReleaseNotes/v1.7.0.md:7]: jist => gist

File: app/i18n/docs/en/ReleaseNotes/v1.7.0.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+This release of Decrediton includes numerous bug fixes and refinement across
+all pages/tabs.
+
+
+* Matheusd has overhauled the security of Decrediton so that we can
```
Comment (matheusd) [inline app/i18n/docs/en/ReleaseNotes/v1.7.0.md:5]: I feel like naming specific authors in the release notes is unnecessary.

**Review by matheusd — COMMENTED** [review]
Needs to add an entry on `ReleaseNotes/Form/Form.module.css` for the v1.7.0 image.

**Comment by alexlyp** [comment]
Whoops, hadn't pushed the image changes!  Doing now

**Review by matheusd — APPROVED** [review]
Reads fine now!

**Pull request merged**